### PR TITLE
Modified my .md file.

### PR DESCRIPTION
**Pull request by gyenney**
Added comment to my caption file.

### CODE REVIEW

**Comment by gyenney** [comment]
@beardofedu  -- forgot to mention you in my original pull request.

**Comment by beardofedu** [comment]
@gyenney Nice! We will talk about Pull Request reviews next class and get this PR merged!

File: _posts/0000-01-03-gyenney.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![panda-birthday](https://cloud.githubusercontent.com/assets/16547949/25400994/60f64804-29c2-11e7-8a6d-4401d7d294b9.jpg)
 {: .center}
 
-Caption here
+Don't poke the bear, especially when he's eating his birthday cake!
```
Comment (crisslyjames) [inline _posts/0000-01-03-gyenney.md]: Good caption!!

Comment (crisslyjames) [reply _posts/0000-01-03-gyenney.md]: You may want to add markdown to your caption
`Don't poke the bear, especially when he's eating his birthday cake!`

**Review by crisslyjames — CHANGES_REQUESTED** [review]
Caption requires markdown

**Review by beardofedu — APPROVED** [review]
🐼 🚢 👍

**Review by crisslyjames — APPROVED** [review]
🚢 👍

File: _posts/0000-01-03-gyenney.md (hunk 1 above)
Comment (gyenney) [reply _posts/0000-01-03-gyenney.md]: I have now updated the caption with markdown.

Comment (gyenney) [reply _posts/0000-01-03-gyenney.md]: How is it now?

**Comment by gyenney** [comment]
Thanks for your help!

**Comment by gyenney** [comment]
Adding a comment after failed merge.

**Pull request merged**